lib/push.js

```diff
@@ -78,7 +79,17 @@ var reasonToEvents = {
   }
 }
 
-module.exports = function (log, db) {
+module.exports = function (log, db, config) {
```

Comment:
You weren't even a *little bit* tempted to break open the shiny new es6 box of tricks? 😄